docs/maatregelen/3-dat-09-dataminimalisatie.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Beperk de omvang van datasets voor energie-efficiëntie
+id: urn:nl:ak:mtr:dat-09
+toelichting: Houd datasets klein en gericht om onnodige energieconsumptie te voorkomen tijdens de verwerking en opslag van data voor algoritmes.
+levenscyclus:
+- dataverkenning-en-datapreparatie
+onderwerp:  
+- data  
+- duurzaamheid
+rollen:  
+- ontwikkelaar
+- projectleider
+hide:  
+- navigation  
+- toc  
+---
+
+<!-- Let op! onderstaande regel met 'tags' niet weghalen! Deze maakt automatisch de knopjes op basis van de metadata  -->
+<!-- tags -->
+
+## Maatregel
+Houd datasets klein en gericht om onnodige energieconsumptie te voorkomen tijdens de verwerking en opslag van data voor algoritmes. We noemen dit ook wel dataminimalisatie.
+
+## Toelichting
+Hoe meer je bewaart, hoe meer ruimte dat kost om op te slaan. Bovendien verbruikt elk apparaat dat nodig is om data op te slaan stroom. Dat heeft grote invloed op de CO₂-uitstoot van een datacentrum.
+Grote datasets brengen daarom hoge energie- en opslagkosten met zich mee. Door de dataset bewust te beperken tot relevante gegevens, kun je ook de energie-efficiëntie van algoritmes aanzienlijk verbeteren. Vooral bij de ontwikkeling van AI-systemen kan het verminderen van data bijdragen aan lagere energiebehoeften en CO₂-uitstoot.
+
+### Technieken voor dataminimalisatie
+- **Slimme selectie van trainingsdata**: Gebruik methoden die irrelevante data uit de dataset filteren, zoals dataselectie-algoritmes en sampling-technieken. Door te focussen op relevante data, beperk je de omvang zonder de prestaties van het model te beïnvloeden.
+- **Verwijderen van redundante en dubbele data**: Deduplicatie van data minimaliseert onnodige verwerkingskracht. Door alleen unieke en relevante gegevens op te slaan, wordt de opslagbehoefte verder beperkt.
+- **Opschonen en archiveren van verouderde data**: [Regelmatige archivering](2-owp-08-archiveren-documenten.md) of verwijdering van verouderde data in je dataset zorgt voor een verminderde voetafdruk en verhoogt ook de efficiëntie.
+
+## Bijbehorende vereiste(n)
+<!-- list_vereisten_on_maatregelen_page -->
+
+## Risico
+Zonder dataminimalisatie loopt je organisatie het risico op onnodig hoge energie- en opslagkosten, en een grotere ecologische impact.
+
+## Bronnen
+- [Rijks ICT-dashboard](https://www.rijksictdashboard.nl/duurzaamheid)
```

Comment:
ADR en ARK zullen hier vast ook wat over zeggen?

Reply:
ADR noemt: PRI.5, maar dat is alleen in relatie tot persoonsgegevens: “Het algoritme verwerkt niet meer persoonsgegevens dan noodzakelijk; de verwerkte gegevens zijn proportioneel en substantieel.”

Maar wel een goede om hierbij te noemen denk ik.